.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,122 @@
+# Contributing to OpenIPSL
+
+Thank you for considering investing your time to contribute to OpenIPSL! :tada::+1:
+
+Here, you will find a set of guidelines for contributing to the OpenIPSL, which is hosted in the [OpenIPSL page](https://github.com/OpenIPSL) on GitHub. Many of the instructions in this document are considered to be suggestions rather than strict rules, so please use your best judgement when reading this document and when contributing to OpenIPSL. In addition, you can find some tips and good practices in [GitHub's Contribution Guideline](https://github.com/github/docs/blob/3c5055fc4eecf3f83560419d94be54458c2f56a1/CONTRIBUTING.md) and in [Atom's Contribution Guideline](https://github.com/atom/atom/blob/master/CONTRIBUTING.md).
```

Comment:
```suggestion
Here, you will find a set of guidelines for contributing to the OpenIPSL, which is hosted on the [OpenIPSL page](https://github.com/OpenIPSL/OpenIPSL) on GitHub. Many of the instructions in this document are considered to be suggestions rather than strict rules, so please use your best judgement when reading this document and when contributing to OpenIPSL. In addition, you can find some tips and good practices in [GitHub's Contribution Guideline](https://github.com/github/docs/blob/3c5055fc4eecf3f83560419d94be54458c2f56a1/CONTRIBUTING.md) and in [Atom's Contribution Guideline](https://github.com/atom/atom/blob/master/CONTRIBUTING.md).
```